packages/taco/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nucypher/taco",
-  "version": "0.1.0",
+  "version": "0.2.2",
```

Comment:
Why 0.2.2 and not 0.2.0?

Reply:
I messed up and published a bad build. I yanked previous versions, but `npmjs.com` doesn't allow republishing so I had to bump the version.